src/components/Editor.jsx

```diff
@@ -37,44 +37,42 @@ const toolbarButtons = applyFilters( 'wpgraphqlide_toolbar_buttons', {
 } );
 
 export function Editor() {
-	const { query, shouldRenderStandalone, plugins } = useSelect(
-		( select ) => {
-			const wpgraphqlIde = select( 'wpgraphql-ide' );
-			return {
-				query: wpgraphqlIde.getQuery(),
-				shouldRenderStandalone: wpgraphqlIde.shouldRenderStandalone(),
-				plugins: wpgraphqlIde.getPluginsArray(),
-			};
-		}
-	);
+	const query = useSelect( ( select ) => {
+		return select( 'wpgraphql-ide' ).getQuery();
+	} );
+
+	const shouldRenderStandalone = useSelect( ( select ) => {
+		return select( 'wpgraphql-ide' ).shouldRenderStandalone();
+	} );
 
-	const { setDrawerOpen, setQuery } = useDispatch( 'wpgraphql-ide' );
+	const { setDrawerOpen } = useDispatch( 'wpgraphql-ide' );
 
 	return (
-		<GraphiQL
-			query={ query }
-			fetcher={ fetcher }
-			onEditQuery={ ( query ) => setQuery( query ) }
-			plugins={ plugins.length > 0 ? plugins : null }
-		>
-			<GraphiQL.Toolbar>
-				{ Object.entries( toolbarButtons ).map( ( [ key, Button ] ) => (
-					<Button key={ key } />
-				) ) }
-			</GraphiQL.Toolbar>
-			<GraphiQL.Logo>
-				{ ! shouldRenderStandalone ? (
-					<button
-						className="button EditorDrawerCloseButton"
-						onClick={ () => setDrawerOpen( false ) }
-					>
-						X
-						<span className="screen-reader-text">close drawer</span>
-					</button>
-				) : (
-					<span />
-				) }
-			</GraphiQL.Logo>
-		</GraphiQL>
+		<>
+			<GraphiQL query={ query } fetcher={ fetcher }>
+				<GraphiQL.Toolbar>
+					{ Object.entries( toolbarButtons ).map(
+						( [ key, Button ] ) => (
+							<Button key={ key } />
+						)
+					) }
```

Comment:
Does it make sense to also revert this part and let the ToolbarButtons be hard-coded for now as well and we'll add back this bit of pluggability in Q2 as part of our focus on the public APIs?

Reply:
Yes. I am going through now with a 2nd pass and removing some of the newer filters